recipes/ngsld/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.1.1" %}
+{% set sha256 = "54331197066eb680b2f8113fdc7eb89f247ce42176b99f59c6950f97a301550c" %}
+
+package:
+  name: ngsld
+  version: {{ version }}
+
+source:
+  url: https://github.com/fgvieira/ngsLD/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('gcc') }}
+    - zlib
+    - gsl
+  run:
+    - md5sum
```

Comment:
It needs md5sum to run? That's quite unusual.

Reply:
You're right, that was my mistake. I was including dependencies mentioned for auxiliary scripts, but that one is for a test script that doesn't work when installing from source. I've taken it out now, thank you for noticing!